src/views/layouts-wallet/TheSendTransactionLayout.vue

```diff
@@ -1,10 +1,9 @@
 <template>
   <v-row>
-    <v-col cols="9">
+    <v-col cols="8" lg="8">
```

Comment:
so cols = 8 meaning that on all views  it will take 8 cols of 12, you dont have to specify lg = 8, unless you want to change how many column its suppose to take up starting from that break point.

Reply:
Fixed
https://github.com/MyEtherWallet/MyEtherWallet/blob/v6/fix-wallet-layouts/src/views/layouts-wallet/TheSendTransactionLayout.vue